### DISCUSSION TITLE
Is it possible to import certificates from certbot configuration?

### DISCUSSION THREAD

**Discussion by synoniem** [go-acme/lego#1723] [Q&A]
I do have several servers with their certificates created and maintained with certbot http challenge. Now I want to migrate to dns challenge with lego but do not see a way to import or copy the current certificates and keys. Running the docker version with a volume linked to /etc/letsencrypt/live/server-fqdn will not be recognised. Linked to /.lego neither. Is there a way make this work?

**Comment by ldez** [comment]
Hello,

we don't have an "import" command (it's a bit weird to say "import" in this context, but I think that I understand the meaning).

I can try to describe the expected file structure if needed.

I think a better and simplest way is just to recreate your certificates with lego (take care of the rate limit of Let's Encrypt)

**Reply by synoniem** [reply]
If you can describe the expected file structure, especially the docker version, it would be very helpfull!
